### PR TITLE
docs(srv): add reference content on sandboxes for func and containers MTA-3569

### PR DESCRIPTION
**Pull request by SamyOubouaziz**
### Your checklist for this pull request

- [x] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: macros/serverless/serverless-sandbox.mdx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+macro: sandbox
+---
+
+This page contains useful information on sandboxing in the context of Scaleway Serverless functions and Serverless Containers.
```
Comment (bene2k1) [inline macros/serverless/serverless-sandbox.mdx:5]:
```suggestion
This page contains useful information on sandboxing in the context of Scaleway Serverless Functions and Serverless Containers. 
```

File: macros/serverless/serverless-sandbox.mdx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+macro: sandbox
+---
+
+This page contains useful information on sandboxing in the context of Scaleway Serverless functions and Serverless Containers. 
+[Sandboxes](serverless/concepts/#sandbox) are an isolation mechanism that allow you to test or execute code without risking harm to the host environment.
+Serverless Functions use two types of sandboxes, [GVisor](https://gvisor.dev/docs/) and [Kata](https://katacontainers.io/learn/).
+
+## GVisor	
+
+OS Level Isolation - Faster cold start but some incompatibility, ideal for API development or web application, works for most workloads.
```
Comment (bene2k1) [inline macros/serverless/serverless-sandbox.mdx:11]:
```suggestion
OS Level Isolation - Faster cold start but some incompatibility, ideal for API development or web applications, works for most workloads.
```

File: macros/serverless/serverless-sandbox.mdx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+GVisor is an application kernel, written in Go, that implements a substantial portion of the Linux system call interface. It provides an additional layer of isolation between running applications and the host operating system.
+
+GVisor is a Linux-compatible sandbox that intercepts the hosted application system calls to the kernel.
+
+GVisor Compatibility
+
+    System Calls working with GVisor: https://gvisor.dev/docs/user_guide/compatibility/linux/amd64/
```
Comment (bene2k1) [inline macros/serverless/serverless-sandbox.mdx:18]:
```suggestion
    [System Calls working with GVisor](https://gvisor.dev/docs/user_guide/compatibility/linux/amd64/)
```

File: macros/serverless/serverless-sandbox.mdx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    Application compatibility with GVisor: https://gvisor.dev/docs/user_guide/compatibility/
+
+Refer to the [official GVisor documentation](https://gvisor.dev/docs/) for more information.
+
+## Kata Container	
+
+VM Level Isolation - Higher cold start, full linux compatibility, better if you need specific system librairies or perform network intensive tasks, not recommended for lower tiers (less than 1 vCPU).
```
Comment (SamyOubouaziz) [inline macros/serverless/serverless-sandbox.mdx:25]:
```suggestion
VM Level Isolation - Slower cold start, full linux compatibility, better if you need specific system librairies or perform network intensive tasks, not recommended for lower tiers (less than 1 vCPU).
```

**Comment by SamyOubouaziz** [comment]
Closing this pull request as specs are outdated.

**Pull request closed without merging**